react/features/base/config/functions.js

```diff
@@ -58,6 +58,21 @@ const WHITELISTED_KEYS = [
      */
     'callUUID',
 
+    /**
+     * The UUID of the CallKit call representing the conference/meeting
```

Comment:
The comments is incorrect: it says `callHandle` is the UUID of the CallKit call.